package chaos_data; message SymbolDynamMarket {
  required uint64 current_time = 1; //成交时间
  required uint32 open_price = 2; //开盘价
  required uint32 high_price = 3; //最高价
  required uint32 low_price = 4; //最低价
  required uint32 new_price = 5; //最新价
  required uint32 volume = 6; //成交量
  required uint64 amount = 7; //成交额
  required uint64 inner_vol = 8;//内盘成交量
  required uint32 tick_count = 9;//累计成交笔数
  repeated uint32 buy_price = 10; //委买价
  repeated uint32 buy_vol = 11;//委买量
  repeated uint32 sell_price = 12;//委卖价
  repeated uint32 sell_vol = 13;//委卖量
  required uint32 open_interest = 30;//持仓量，期货期指特有
  required uint32 settle_price = 31;//结算价 期货期指现货特有
}
